var/spack/repos/builtin/packages/glib/package.py

```diff
@@ -32,18 +32,26 @@ class Glib(Package):
        and interfaces for such runtime functionality as an event loop,
        threads, dynamic loading and an object system."""
     homepage = "https://developer.gnome.org/glib/"
-    url = "http://ftp.gnome.org/pub/gnome/sources/glib/2.42/glib-2.42.1.tar.xz"
+    url      = "http://ftp.gnome.org/pub/gnome/sources/glib/2.42/glib-2.42.1.tar.xz"
 
+    version('2.49.4', 'e2c87c03017b0cd02c4c73274b92b148')
     version('2.42.1', '89c4119e50e767d3532158605ee9121a')
 
-    depends_on("libffi")
-    depends_on("zlib")
-    depends_on("pkg-config", type='build')
+    depends_on('libffi')
+    depends_on('zlib')
+    depends_on('pkg-config', type='build')
     depends_on('gettext', when=sys.platform == 'darwin')
+    depends_on('pcre+utf', when='@2.49:')
+    depends_on('gettext', when='@2.49:')
```

Comment:
@becker33 I couldn't get this syntax to work for a `depends_on` command and I couldn't find any similar example in the current source code. If you can show me how to implement this conditional I will include it.